content/blog/2025/oddcontrast-guide.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Make the Web a More a Colorful Place!
+sub: A guide to using new color spaces & formats with OddContrast
+author: oddbird
+date: 2025-06-05
+tags:
+  - Article
+  - Accessibility
+  - Color
+  - Design
+  - OddContrast
+  - OddTools
+image:
+  src: blog/2025/oddcontrast-guide.jpg
+  alt: >
+    OddContrast displays P3 gamut range
+    in Oklch color format
+summary: |
+  OddBird's color tool not only checks contrast ratios,
+  but supports the new CSS color formats and spaces.
+---
+
+{% import 'embed.macros.njk' as embed %}
+
+For years designers and developers were limited to colors in the sRGB colors
+space, using formats like Hexadecimal, RGB, and HSL. As display technology
+progressed, so too has CSS, and we have access to additional color spaces and
+wider gamuts. These advances led us to build OddContrast, a color editing and
+testing tool that handles our new world of modern color formats.
+
+## The flexibility of OddContrast
+
+Although it looks simple on the surface, OddContrast has many built-in
+conveniences when working with color.
+
+The default color format is display-p3 but you can select from the following
```

Comment:
```suggestion
The default color format is `display-p3`, but you can select from the following
```

Reply:
updated in 613995ff159685af68a52ea345394d3d1f006710